WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaEvents.java

```diff
@@ -0,0 +1,51 @@
+package org.wordpress.android.ui.media.services;
+
+public class MediaEvents {
+    public static class MediaUploadSucceed {
+        public final String mLocalBlogId;
+        public final String mLocalMediaId;
+        public final String mRemoteMediaId;
+        MediaUploadSucceed(String localBlogId, String localMediaId, String remoteMediaId) {
+            mLocalBlogId = localBlogId;
+            mLocalMediaId = localMediaId;
+            mRemoteMediaId = remoteMediaId;
+        }
+    }
+
+    public static class MediaUploadFailed {
+        public final String mLocalMediaId;
+        public final String mErrorMessage;
+        MediaUploadFailed(String localMediaId, String errorMessage) {
+            mLocalMediaId = localMediaId;
+            mErrorMessage = errorMessage;
+        }
+    }
+
+    public static class MediaAdded {
```

Comment:
Looking at the code, separate events for `MediaAdded`, `MediaFetched` and `MediaStateChanged` aren't necessary. A single `MediaChanged` event could replace all three of these.